Package.swift

```diff
@@ -531,7 +531,26 @@ let package = Package(
         .linkedFramework("SystemConfiguration", .when(platforms: [.iOS, .macOS, .tvOS])),
       ]
     ),
-
+    .testTarget(
+      name: "FirebaseCrashlyticsUnit",
+      dependencies: ["FirebaseCrashlytics", "OCMock"],
+      path: "Crashlytics/UnitTests",
+      resources: [
+        .process("FIRCLSMachO/data"),
+        .copy("Data"),
```

Comment:
Retains the directory’s structure since in there are files have same names under this directory. Example: `metadata.clsrecord`
<img width="375" alt="Screen Shot 2022-11-21 at 9 55 54 AM" src="https://user-images.githubusercontent.com/16548721/203086199-96a990c6-d359-4751-b861-3175794f8c74.png">